.github/ISSUE_TEMPLATE.md

```diff
@@ -1,3 +1,4 @@
+<!--
 If you have a question please search or post to our Discourse site: https://discourse.julialang.org.
```

Comment:
comma after "question"

Reply:
I'd rather not pollute this PR with changing the actual text, since it wasn't written by me. I just wanted to put the whole text inside a comment.